narwhals/_dask/namespace.py

```diff
@@ -135,6 +137,52 @@ def any_horizontal(self, *exprs: IntoDaskExpr) -> DaskExpr:
     def sum_horizontal(self, *exprs: IntoDaskExpr) -> DaskExpr:
         return reduce(lambda x, y: x + y, parse_into_exprs(*exprs, namespace=self))
 
+    def concat(
+        self,
+        items: Iterable[DaskLazyFrame],
+        *,
+        how: str = "vertical",
+    ) -> DaskLazyFrame:
+        import dask.dataframe as dd  # ignore-banned-import
+
+        if len(list(items)) == 0:
+            msg = "No items to concatenate"
+            raise AssertionError(msg)
+        native_frames = [i._native_frame for i in items]
+        axis: int
+        join: str
+        if how == "vertical":
+            if not all(
+                tuple(i.columns) == tuple(native_frames[0].columns) for i in native_frames
+            ):  # pragma: no cover
+                msg = "unable to vstack with non-matching columns"
+                raise AssertionError(msg)
+            axis = 0
+            join = "inner"
+        elif how == "horizontal":
+            all_column_names: list[str] = list(chain(*[i.columns for i in native_frames]))
```

Comment:
Hi @benrutter, I believe this can be "simplified" like this, but feel free to keep it as it is though, just my 2 cents

```suggestion
            all_column_names: list[str] = [column for frame in native_frames for column in frame.columns]
```

Reply:
Thanks @aidoskanapyanov! That definitely is simpler to read - I'm on a mobile at the moment so will merge in later 😁 (think I need to take out then itertools import as well or it'll fail linting).

Reply:
I'm guessing pre-commit CI is going to remove the import via autofix 😄